src/sqlancer/feldera/FelderaSchema.java

```diff
@@ -121,6 +81,31 @@ public static FelderaDataType getRandomNonNullType() {
         public static FelderaDataType getRandomType() {
             return Randomly.fromOptions(values());
         }
+    }
+
+    public static class FelderaCompositeDataType {
```

Comment:
this is not general enough, MAP has two generic arguments.

Reply:
Maybe call this GenericType?
Too bad we aren't reusing the structures we already have in the compiler.